drafts/draft-20005.adoc

```diff
@@ -0,0 +1,223 @@
+---
+layout: post
+title: "Updates to JAAS Custom Login Modules are now available on Open Liberty 20.0.0.5"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/tomjenningss
+author_github: https://github.com/tomjenningss
+seo-title: Open Liberty 20.0.0.5 provides enhanced, updated features to JAAS Custom Login Modules which means you can now use JAAS Custom Login Modules that are packaged within resource adapters on Open Liberty 20.0.0.5.
+seo-description: Open Liberty 20.0.0.5 provides enhanced, updated features to JAAS Custom Login Modules which means you can now use JAAS Custom Login Modules that are packaged within resource adapters.
+blog_description: Open Liberty 20.0.0.5 provides enhanced, updated features to JAAS Custom Login Modules which means you can now use JAAS Custom Login Modules that are packaged within resource adapters.
+---
+= Updates to JAAS Custom Login Modules are now available on Open Liberty 20.0.0.5
+Tom Jennings <https://github.com/tomjenningss>
+:imagesdir: /
+:url-prefix:
+:url-about: /about/
+
+// tag::intro[]
+Open Liberty 20.0.0.5 provides enhanced, updated features to JAAS Custom Login Modules which means you can now use JAAS Custom Login Modules that are packaged within resource adapters.
+
+In link:{url-about}[Open Liberty] 20.0.0.5:
+
+* <<CLM, JAAS Custom Login Modules>>
+* <<OLL, Open Liberty Logging>>
+* <<JLO, Omit specified fields from JSON logging output>>
+* <<EPT, EJB Persistent Timers>>
+
+View the list of fixed bugs in https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A20004+label%3A%22release+bug%22+[20.0.0.5].
```

Comment:
like should be `https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A20005+label%3A%22release+bug%22+`

Reply:
looks like the text of the link `[20.0.0.5]` got dropped when the link was updated.
This should read 
View the list of fixed bugs in https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A20005+label%3A%22release+bug%22+[20.0.0.5].